src/dataflow-types/types.rs

```diff
@@ -552,6 +552,20 @@ pub struct TailSinkConnector {
     pub since: Timestamp,
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub enum SinkConnectorBuilder {
+    Kafka(KafkaSinkConnectorBuilder),
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct KafkaSinkConnectorBuilder {
+    pub broker_url: Url,
+    pub schema_registry_url: Url,
+    pub value_schema: String,
+    pub topic_prefix: String,
+    pub topic_suffix: String,
```

Comment:
maybe worth some kind of comment here about how the suffix is designed to be distinct across restarts?

Reply:
I'll get to this in a follow-up PR. (I also don't think here is quite the right spot, because dataflow_types doesn't care at all what properties the suffix has.)